apps.yml

```diff
@@ -4182,3 +4182,15 @@ apps:
     url: https://auth.mozilla.auth0.com/samlp/TPliDroeFM4p3csAsmC0tfF8AxeOisrP
     vanity_url:
     - /compass
+- application:
+    authorized_groups:
+      - mozilliansorg_netsuite-access
```

Comment:
If you're feeling linty, dedent this a couple of spaces

Reply:
Oh good catch, stupid text editor.